customization/customizing-the-backend.md

```diff
@@ -65,59 +64,61 @@ Our new business logic is pretty easy to test in integration:
 
 {% code title="spec/models/spree/order\_spec.rb" %}
 ```ruby
+require 'rails_helper'
+require 'amazing_store/order_analyzer'
```

Comment:
I'd like to explore with you why we need to require this module manually. It shouldn't be needed.

Reply:
This should not be required anymore after loading the whole `/lib` folder in the `eager_load_paths`, isn't it?

Reply:
Moved files to app/models as discussed